### PR TITLE
Implement Graphcool schema

### PR DESCRIPTION
**Pull request by alexfts**
Add all major types to graphcool

Add most important relations

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "name": "server",
   "version": "1.0.0",
-  "description": "My Graphcool Service"
-}
\ No newline at end of file
+  "description": "My Graphcool Service",
```
Comment (siddharthparmar7) [inline server/package.json]: Please provide a better description.

File: server/types.graphql [hunk 2]
```diff
@@ -1,23 +1,86 @@
...
-  dateOfBirth: DateTime
+  email: String! @isUnique
+  password: String! @isUnique
+  dateOfBirth: DateTime!
+  name: String!
+  points: Int! @default(value: 0)
+  profilePicture: File @relation(name: "UserPicture")
+  cardBarcode: String
+  favouriteBeers: [Beer!]! @relation(name: "UserBeers")
+  pointsHistory: [Point!]! @relation(name: "UserPoint")
```
Comment (siddharthparmar7) [inline server/types.graphql]: did you mean to say ```@relation(name: UserPoints)``` ?

File: server/types.graphql [hunk 3]
```diff
@@ -1,23 +1,86 @@
...
+  name: String!
+  url: String!
+  contentType: String!
+  user: User! @relation(name: "UserPicture")
+}
+
+type Point @model {
+  id: ID! @isUnique
+  user: User! @relation(name: "UserPoint")
+  beer: Beer! @relation(name: "BeerPoint")
```
Comment (siddharthparmar7) [inline server/types.graphql]: BeerPoints? 
UserPoints?

these are one-many relationships, be careful with your naming convention.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice Work! please make the requested changes.

File: server/package.json (hunk 1 above)
Comment (alexfts) [reply server/package.json]: Done!

File: server/types.graphql (hunk 2 above)
Comment (alexfts) [reply server/types.graphql]: Fixed

File: server/types.graphql (hunk 3 above)
Comment (alexfts) [reply server/types.graphql]: Fixed

**Comment by alexfts** [comment]
Fixed!

**Review by siddharthparmar7 — APPROVED** [review]
Nice work! Thank you!

**Pull request merged**